community/joe/depends

```diff
@@ -0,0 +1,3 @@
+make
+autoconf
```

Comment:
autoconf is not required for `./configure; make; make install`. It's only required if you're calling `autoreconf` or `autogen.sh`